documentation/docs/tirith-policies/tirith-policy-variables.md

```diff
@@ -35,4 +35,32 @@ import TabItem from '@theme/TabItem';
       "item": "https://www.lambdatest.com/support/docs/tirith-policy-variables/"
     }]
   })
-}}></script>
\ No newline at end of file
+}}></script>
+Policy variables allow dynamic values in policy definitions. They can be referenced in conditions to make policies more flexible.
+
+```javascript title="policy.json"
+{
+  "meta": {
+    "version": "v1",
+    "variables": {
```

Comment:
Please remove this, and move it into a new code section titled `variables.json`



variables.json
```
{ "max_epoch" ....
```

example command
```
tirith -input-path <INPUT_PATH> -policy-path policy.json -var-path variables.json
```